features/rni.feature

```diff
@@ -0,0 +1,202 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
+    - RNI-deelnemer gegevens worden ook geleverd wanneer hier niet naar gevraagd is met de fields parameter
+    - categorie wordt gevuld met de naam van categorie waarop de RNI-deelnemer gegevens heeft aangeleverd
+      | categorie | naam           |
+      | 01        | Persoon        |
+      | 04        | Nationaliteit  |
+      | 06        | Overlijden     |
+      | 07        | Inschrijving   |
+      | 08        | Verblijfplaats |
+
+
+    Achtergrond:
+      Gegeven landelijke tabel "RNI-deelnemerstabel" heeft de volgende waarden
+      | code | omschrijving                                      |
+      | 101  | Belastingdienst (inzake heffingen en toeslagen)   |
+      | 201  | Sociale Verzekeringsbank (inzake AOW, Anw en AKW) |
+
+
+    Scenario: RNI-deelnemer op geleverd gegeven
+      Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
Deze 'Gegeven' en de daarop volgende 'En' step leidt bij mij tot een fout.  Samenvoegen van beide steps en de tabellen omzetten naar een horizontale variant geeft mij mij wel een succesvol resultaat, dus:

      Gegeven een persoon heeft de volgende 'persoon' gegevens
      | burgerservicenummer (01.20) | RNI-deelnemer (88.10) | Omschrijving verdrag (88.20)                | voornamen (02.10) |       
	  | 000000012                   | 101                   | Belastingverdrag tussen België en Nederland | Peter             |

Reply:
Moeten we dus alle persoonsgegevens altijd in exact 1 stap definiëren? In andere features komt dit heel vaak voor.

Reply:
Ik vind "Gegeven een persoon heeft de volgende 'persoon' gegevens" wel raar dubbelop.

Waarom niet "Gegeven een persoon met burgerservicenummer '000000012' heeft de volgende gegevens"?

Reply:
Blijkbaar komt dit omdat de code die verticale tabellen ondersteund nog niet ingecheckt is.
Ik vermoed dat zodra dat wel is gebeurd beide tabellen nog steeds in elkaar geschoven moeten worden en de 'Gegeven' step moet worden gewijzigd naar:

`Gegeven een persoon heeft de volgende 'persoon' gegevens`

Reply:
aangepast naar Gegeven de persoon met burgerservicenummer '<burgerservicenummer>' heeft de volgende gegevens

volgens mij is deze equivalent aan "Gegeven een persoon heeft de volgende 'persoon' gegevens" plus het opgeven van burgerservicenummer. Lees n.m.m. veel beter